booker_winners.sql

```diff
@@ -0,0 +1,60 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 5446
+#
+# https://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.33)
+# Database: booker_winners
+# Generation Time: 2021-03-29 10:00:08 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+SET NAMES utf8mb4;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table booker_winners
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `booker_winners`;
+
+CREATE TABLE `booker_winners` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `prize_year` int(11) unsigned DEFAULT NULL,
```

Comment:
suggest using the YEAR datatype here (With a length of 4)

You should be able to change it without having to update your data!